modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/portlet/action/ViewListTypeDefinitionsMVCRenderCommand.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.web.internal.object.definitions.portlet.action;
+
+import com.liferay.list.type.model.ListTypeDefinition;
+import com.liferay.object.constants.ObjectPortletKeys;
+import com.liferay.object.web.internal.list.type.display.context.ViewListTypeDefinitionsDisplayContext;
+import com.liferay.portal.kernel.portlet.bridges.mvc.MVCRenderCommand;
+import com.liferay.portal.kernel.security.permission.resource.ModelResourcePermission;
+import com.liferay.portal.kernel.util.Portal;
+import com.liferay.portal.kernel.util.WebKeys;
+
+import javax.portlet.PortletException;
+import javax.portlet.RenderRequest;
+import javax.portlet.RenderResponse;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Selton Guedes
+ */
+@Component(
+	property = {
+		"javax.portlet.name=" + ObjectPortletKeys.LIST_TYPE_DEFINITIONS,
+		"mvc.command.name=/object_definitions/view_list_type_definitions"
```

Comment:
I believe the value of this variable should be: "mvc.command.name=**/list_type_definitions**/view_list_type_definitions"
Besides that, this class should be in `com.liferay.object.web.internal.list.type.portlet.action`.

Reply:
Hi, @marcelabc. I saw that we don't need this class because the ListTypeDefinitionsPortlet is being called through [here](https://github.com/liferay-objects/liferay-portal/pull/2885/commits/2fdec055834123268250f1e616a01d045dce9d3f#diff-64c363b3b1041b2ef838f5dc1cd27383d70d38b37b78dbac633db8ad7f604980R45-R50). This Portlet is setting the attributes that we need to render the `/list_type_definitions/view.jsp`